src/multizimbutton.cpp

```diff
@@ -10,16 +10,27 @@
 MultiZimButton::MultiZimButton(QWidget *parent) :
     QToolButton(parent), 
     mp_buttonList(new QListWidget),
-    mp_radioButtonGroup(new QButtonGroup(this))
+    mp_radioButtonGroup(new QButtonGroup(this)),
+    mp_selectAllButton(new QCheckBox(gt("select-all"), this))
 {
     setMenu(new QMenu(this));
     setPopupMode(QToolButton::InstantPopup);
     setDefaultAction(KiwixApp::instance()->getAction(KiwixApp::OpenMultiZimAction));
     connect(this, &QToolButton::triggered, this, &MultiZimButton::showMenu);
 
-    const auto popupAction = new QWidgetAction(menu());
-    popupAction->setDefaultWidget(mp_buttonList);
-    menu()->addAction(popupAction);
+    /* TODO: connect with list widget after 'real' multi-zim implementation */
+    mp_selectAllButton->setDisabled(true);
+    mp_selectAllButton->hide();
```

Comment:
Frankly, I don't think that it makes sense to include an invisible "Select all" button in this PR.

Reply:
I believe Kelson just wants this UI to be complete before the actual multi-Zim is implemented.

Reply:
I am not saying that this code should be deleted and forgotten. Please move it in a separate draft PR.

Reply:
Done #1234